mk/main.mk

```diff
@@ -20,7 +20,9 @@ CFG_RELEASE_NUM=1.1.0
 # versions (section 9)
 CFG_PRERELEASE_VERSION=.1
 
-CFG_FILENAME_EXTRA=4e7c5e5c
+# Append a version-dependent hash to each library, so we can install different
+# versions in the same place
+CFG_FILENAME_EXTRA=$(shell echo -n $(CFG_RELEASE) | $(CFG_HASH_COMMAND))
```

Comment:
I don't remember the details but `echo -n` can't be counted on to behave the same everywhere. Can you change this to just `echo`? *Hopefully* that results in the same output everywhere - we need these hashes to come out the same.